recipes/tianshou/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "tianshou" %}
+{% set version = "0.3.0rc0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/thu-ml/tianshou/releases/download/v{{ version }}/tianshou-{{ version }}.tar.gz
+  sha256: 78403c7b66c11165460d244811573dd640abcde4e322f79fe4a35dc89602329c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - gym >=0.15.4
+    - tqdm
+    - tensorboard
+    - pytorch >=1.4.0
+    - numba >=0.51.0
+
+test:
+  imports:
+    - tianshou
+
+about:
+  home: https://github.com/thu-ml/tianshou
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Please add a comment above the `license_file` line that says something like:

```
# License file manually packaged. See https://github.com/thu-ml/tianshou/pull/227`
```

I opened up a PR (https://github.com/thu-ml/tianshou/pull/227) to include the license file in the source distribution. If that PR is merged or some other solution is implemented, then in future releases, this acts as a reminder to use the copy of the license file that comes in the source distribution.

Reply:
```suggestion
    # License file manually packaged. See https://github.com/thu-ml/tianshou/pull/227
    license_file: LICENSE
```

Reply:
Should I remove the LICENSE in this PR?